### ISSUE TITLE
What is "recompilation profiler" in doc? (Seems to have a dangling link)

### ISSUE THREAD

**Issue by fzyzcjy** [pytorch/pytorch#141422]
### 📚 The doc issue

https://pytorch.org/docs/stable/torch.compiler_faq.html says:

![image](https://github.com/user-attachments/assets/83bd3f29-6ed2-4ce3-b93f-65cc25ed412c)

But by clicking on it, it jumps to nowhere. I would appreciate it if I could know how to debug this excessive recompilation issue.

**Comment by yf225** [comment]
cc. @ezyang curious would OSS tlparse be helpful for this?

**Comment by ezyang** [comment]
yes, you should use TORCH_TRACE instead, this is out of date, see https://docs.google.com/document/d/1y5CRfMLdwEoF1nTk9q8qEu1mgMUuUtvhklPKJ2emLU8/edit?tab=t.0#heading=h.t130sdb4rshr

**Comment by fzyzcjy** [comment]
Thank you!
